Ginger/GingerTest/GingerAutomator.cs

```diff
@@ -49,18 +49,26 @@ public class GingerAutomator
         public static bool Highlight { get { return false; }  }
 
         public static GingerAutomator StartSession()
-        {            
-            SessionCount++;
-            TestMutex.WaitOne();  // Make sure we run one session at a time, wait for session to be free
-            if (app == null)
+        {
+            try
             {
-                gingerAutomatorInstance = new GingerAutomator();
-                gingerAutomatorInstance.StartGinger();
-                while (!isReady)
+                SessionCount++;
+                TestMutex.WaitOne();  // Make sure we run one session at a time, wait for session to be free
+                if (app == null)
                 {
-                    Thread.Sleep(100);
+                    gingerAutomatorInstance = new GingerAutomator();
+                    Console.WriteLine("Starting Ginger UI");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)